app/services/sufia/collection_size_service.rb

```diff
@@ -0,0 +1,34 @@
+# returns a list of solr documents for collections the item is a part of
+module Sufia
+  class CollectionSizeService
+    include Blacklight::Configurable
+    include Blacklight::SearchHelper
+
+    attr_reader :collection
+
+    self.copy_blacklight_config_from(CatalogController)
+
+    def self.run(collection)
+      new(collection).collection_size
+    end
+
+    def initialize(collection)
+      @collection = collection
+    end
+
+    def collection_size()
+      query = collection_search_builder.rows(1000).merge({ fl:[size_field]})
```

Comment:
What does the hard-coded `1000` represent?  Should this be a constant with a descriptive name, so folks know what it is and can override it?